packages/cashier/src/containers/routes/routes.jsx

```diff
@@ -1,34 +0,0 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import { withRouter } from 'react-router';
-import { connect } from 'Stores/connect';
-import BinaryRoutes from './binary-routes.jsx';
-import ErrorComponent from './error-component';
-
-const Routes = ({ error, has_error, is_logged_in, is_logging_in, passthrough }) => {
-    if (has_error) {
```

Comment:
remove `passthrough` property because we don't use it in BinaryRoutes and RouteWithSubRoutes now